index.js

```diff
@@ -40,24 +55,41 @@ module.exports = {
     function getBytesStored(feedId, cb) {
       /** @type {IndexPlugin} */
       const indexPlugin = ssb.db.getIndex('storageUsed')
-      ssb.db.onDrain('storageUsed', () => {
-        cb(null, indexPlugin.getBytesStored(feedId))
+      stats((err, info) => {
+        if (err) return cb(err)
+        ssb.db.onDrain('storageUsed', () => {
+          const logBytes = indexPlugin.bytesStored.get(feedId)
+          if (logBytes == null) return cb(null, 0)
+          cb(null, sumLogAndIndexes(logBytes, info))
+        })
       })
     }
 
     /**
      * @param {import('./types/helpers').CB<*>} cb
      */
     function stats(cb) {
-      ssb.db.onDrain('storageUsed', () => {
-        getStats(ssb, config.path, cb)
-      })
+      getStats(ssb, config.path, cb)
     }
 
     function stream() {
       /** @type {IndexPlugin} */
       const indexPlugin = ssb.db.getIndex('storageUsed')
-      return indexPlugin.stream()
+      const source = deferred.source()
```

Comment:
`pull-defer` probably seems difficult to use/read, but what it does is simple: it allows us to synchronously return a pull-stream within this function, but *asynchronously* "populate" the stream.